pypeit/core/flexure.py

```diff
@@ -314,8 +316,8 @@ def spec_flexure_obj(specobjs, maskslits, method, sky_file, mxshft=None):
     # Load Archive
     sky_spectrum = load_sky_spectrum(sky_file)
 
-    nslits = len(maskslits)
-    gdslits = np.where(~maskslits)[0]
+    nslits = len(bpm)
+    gdslits = np.where(np.invert(bpm))[0]
```

Comment:
I've generally used `np.logical_not` instead of `np.invert`... it's safer, but you're welcome to disagree!

Reply:
Was unaware of `np.logical_not`.

I agree we should move that way and fear 
we have `invert` throughout the code right now..